### DISCUSSION TITLE
bufsize的使用问题

### DISCUSSION THREAD

**Discussion by pmkol** [IrineSistiana/mosdns#391] [Q&A]
在文档中看到了bufsize的使用方式：
请问bufsize是限制访问上游DNS时应使用的参数？
还是用户访问server时应使用的参数？

另外当我没有引入bufsize插件时，是默认限制512么？
还麻烦作者解答下，建议把使用案例更新在WIKI里。

**Comment by IrineSistiana** [comment]
bufsize 只是修改请求里 edns0 的 udpsize，保证其不会过大。

edns0 比较复杂一两句说不清。建议 Google edns0 udpsize。

**Reply by pmkol** [reply]
> bufsize 只是修改请求里 edns0 的 udpsize，保证其不会过大。
> 
> edns0 比较复杂一两句说不清。建议 Google edns0 udpsize。

相关知识已经查询过了，主要是想了解这个参数使用位置。

可能是我之前没描述清楚，我想了解的是：
这个参数
是用来限制 请求上游DNS时的udpsize
还是用来限制 用户请求mosdns时的udpsize

**Reply by IrineSistiana** [reply]
谁都不限制，这就是个指示性参数。
